.github/workflows/main.yaml

```diff
@@ -133,3 +133,16 @@ jobs:
         run: |
           export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
           make test
+
+  build-image:
+    needs:
+      - setup
+    runs-on: ubuntu-latest
```

Comment:
Should this be another workflow? And only run it if pkg/cleaner was modified or cmd/cleaner ..?

Should be doable by setting it in the on<pull_request>.paths